### ISSUE TITLE
Custom gran prix does not show all tracks

### ISSUE THREAD

**Issue by ricardoV94** [supertuxkart/stk-code#4325]
I tried to create a Custom Gran Prix for splitscreen multiplayer and added 8 tracks. However, when selecting it in the splitscreen multiplayer menu it only showed the first 4 tracks. I then reduced the number of tracks to 4, and in the splitscreen multiplayer menu it showed only 2. The Custom prix included a mix of standard and addon tracks.

STK release version: 1.1
STK Source: Ubuntu Software Store 
System: Ubuntu 20.04

**Comment by Benau** [comment]
can only reproduce once... try clicking the track category below and reclick gp..

**Comment by ricardoV94** [comment]
Still have this problem in version 1.2. I crate a custom GP with several tracks and then when I select it in the local multiplayer only a subset of the tracks are shown (6/2, last time I tried it).

![image](https://user-images.githubusercontent.com/28983449/97088448-34cd2400-1631-11eb-83a4-0144fcace61d.png)

![image](https://user-images.githubusercontent.com/28983449/97088471-4adae480-1631-11eb-9d0d-4905744d80bc.png)

**Comment by Benau** [comment]
Seem that it's hidden because you have not unlocked those tracks yet, try use <unlock_everything value="2" /> in config.xml

**Comment by Benau** [comment]
seems that
```
        return (!m_editable ||
                !PlayerManager::get()->getCurrentPlayer()->isLocked(id));
```
in ```bool GrandPrixData::isTrackAvailable``` is wrong...., !m_editable is not needed... (or without !)??

or ![](https://cdn.lihkg.com/assets/faces/mouse/smile.gif), auria, alayan!

**Comment by cap-42** [comment]
I'm having exactly the same issue.
It's always the same tracks that are missing from the grand prix. Besides that, it seems rather random as both addon and standard tracks are excluded when starting the grand prix.

**Comment by Benau** [comment]
should be fixed by #4561 ![](https://cdn.lihkg.com/assets/faces/pig/clap.gif)
